minutes/base.py

```diff
@@ -24,6 +23,19 @@ class BaseModel:
         'random_state',
     }
 
+    @property
+    def preprocessing_params(self):
+        """Returns a mapping of parameters that are required to do preprocessing
+        of audio data suitable for this model. Useful as kwargs to audio
+        manipulation classes.
+        """
+        return {
+            i: getattr(self, i) for i in self.intialization_params
+            if i in {
+                'ms_per_observation',
```

Comment:
I think it would be good to declare this set as a constant somewhere and refer to it by name.

Reply:
Of course!